### PR TITLE
CR-1119089 Add init script to inform apu booted successfully

### PR DESCRIPTION
**Pull request by rbramand-xilinx**
> Added changes in build_edge.sh to create a init script that will run after apu boot
> This init script will write to sysfs entry exposed by zocl to indicate apu booted successfully
> Fixed build_edge.sh -full compilation error
> Minor fix: Added -n option to sbuild to print log to stdout

### CODE REVIEW

**Review by chvamshi-xilinx — CHANGES_REQUESTED** [review]
Hi @rbramand-xilinx , Can you please create a file in separate directory and maintain instead of creating file inside script.

File: build/build_edge.sh [hunk 1]
```diff
@@ -105,18 +105,89 @@ config_versal_project()
...
+ 
+[Service]
+ExecStart=/usr/bin/apu-boot
+StandardOutput=journal+console
+ 
+[Install]
+WantedBy=multi-user.target
+EOF
+
+    # Create script to check apu boot
```
Comment (rozumx) [inline build/build_edge.sh]: Format regarding the indention looks weird.

**Review by rozumx — APPROVED** [review]
Looks reasonable.  

Side Note
The formatted indention of this file should be re-examined for it makes the code somewhat obfuscated and difficult to follow the intent.  Being that data files are being produced by it, there might not be anything that can be done.

**Review by chvamshi-xilinx — CHANGES_REQUESTED** [review]
We shouldnt have full rootfs config file here. Just update only configs which u r updating.

**Review by chvamshi-xilinx — APPROVED** [review]
Looks good.
Can you please send this recipe to yocto sothat we dont need to maintain in future.

**Comment by rbramand-xilinx** [comment]
> Looks good. Can you please send this recipe to yocto sothat we dont need to maintain in future.

sure @chvamshi-xilinx

**Review by houlz0507 — APPROVED** [review]

**Comment by dayeh-xilinx** [comment]
retest this please. Fixed the centos8 repo issue.

**Pull request merged**